paddlets/automl/autots.py

```diff
@@ -179,10 +179,10 @@ def __init__(
     def fit(
             self,
             train_tsdataset: Union[TSDataset, List[TSDataset]],
-            valid_tsdataset: Union[TSDataset, List[TSDataset]] = None,
-            n_trials: int = 20,
-            cpu_resource: float = 1.0,
-            gpu_resource: float = 0
+            valid_tsdataset: Optional[Union[TSDataset, List[TSDataset]]] = None,
+            n_trials: Optional[int] = 20,
+            cpu_resource: Optional[float] = 1.0,
+            gpu_resource: Optional[float] = 0
```

Comment:
should Not be Optional, refers to https://docs.python.org/3/library/typing.html#typing.Optional